lib/random-colors.ts

```diff
@@ -40,15 +42,11 @@ function createRandomColor(rng: Random): string {
 
   //just sample sRGB if I couldn't sample a random LUV color
   if (luv_sample_failed) {
-    console.log("Sampling sRGB");
-    let rgb = new Array<number>(3).map(() =>
-      rng.inRange({ min: 0, max: 255, step: 1 })
-    );
-    console.log(rgb);
-    for (let i = 0; i < rgb.length; i++) {
-      rgb[i] = rgb[i] / 255.0;
-    }
-    console.log(rgb);
+    //console.log("Sampling sRGB");
+    let rgb = new Array<number>(3)
```

Comment:
Oh just so you know, from an idiomatic standpoint, it might be easier to write `[0, 0, 0]` instead of `new Array<number>(3).fill(0)` here.